### PR TITLE
Urmat: Homework Generics

### PR DESCRIPTION
**Pull request by uri-bbb**

### CODE REVIEW

File: 02-GenericsAndLambda/src/main/java/homework/urmat/TaskGenerics.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package homework.urmat;
+
+public class TaskGenerics {
+
+    public static void main(String[] args){
+        double length = 3.14;
+        double width = 3.14;
+        double height = 3.14;
+
```
Comment (mirza-05) [inline 02-GenericsAndLambda/src/main/java/homework/urmat/TaskGenerics.java:9]: Remove extra Whitespace

File: 02-GenericsAndLambda/src/main/java/homework/urmat/TaskGenerics.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        Dimension<Double> dimension = new Dimension<>(length,width,height);
+
```
Comment (mirza-05) [inline 02-GenericsAndLambda/src/main/java/homework/urmat/TaskGenerics.java:11]: Remove extra Whitespace

File: 02-GenericsAndLambda/src/main/java/homework/urmat/TaskGenerics.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package homework.urmat;
+
```
Comment (mirza-05) [inline 02-GenericsAndLambda/src/main/java/homework/urmat/TaskGenerics.java:2]: Remove extra Whitespace

File: 02-GenericsAndLambda/src/main/java/homework/urmat/PairTest.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package homework.urmat;
+
```
Comment (mirza-05) [inline 02-GenericsAndLambda/src/main/java/homework/urmat/PairTest.java:2]: Remove extra Whitespace

File: 02-GenericsAndLambda/src/main/java/homework/urmat/TaskGenerics.java (hunk 2 above)
Comment (uri-bbb) [reply 02-GenericsAndLambda/src/main/java/homework/urmat/TaskGenerics.java:9]: Done! Thanks!

**Pull request merged**